packages/bp/src/core/media/providers/ghost-media-service.ts

```diff
@@ -50,9 +50,9 @@ export class GhostMediaService implements MediaService {
     // make sure the file name is a valid URI
     fileName = encodeURIComponent(fileName)
     if (this.botId) {
-      return `/api/v1/bots/${this.botId}/media/${fileName}`
+      return `api/v1/bots/${this.botId}/media/${fileName}`
     } else {
-      return `/api/v1/media/${fileName}`
+      return `api/v1/media/${fileName}`
```

Comment:
This needs more thorough testing. We already have some logic on the studio which handles a custom ROOT_PATH, but we still expect URL to start with /

Reply:
Say we fix validation
any reason we want to expect / ?

Reply:
also I meant to ask, this specific file`ghost-media-service` was changed in both the studio branch and this one.
Is one of them "dead code" or do they both run. The files are exact copies:
studio: `packages/studio-be/src/core/media/providers/ghost-media-service.ts`
botpress: `packages/bp/src/core/media/providers/ghost-media-service.ts`

Reply:
The only place where we need to change the logic is in the utils.js file. When we serve the studio, we update the base HREF of every single URL of the studio, thus every URL must start with /
![image](https://user-images.githubusercontent.com/42552874/140565823-5a668ea1-6cf7-4dbe-aa86-31222193a82a.png)

![image](https://user-images.githubusercontent.com/42552874/140565680-657bf613-48d5-4e19-82fb-83e77d6b3be1.png)